src/menu/pref_menu.js

```diff
@@ -0,0 +1,17 @@
+import settings from "electron-settings";
+
+export const prefMenu = {
+  label: "Preferences",
```

Comment:
Can you change this to `Settings`? (since macOS has a very specific meaning for the word "Preferences" and that item usually lives under the main menu in macOS, which could confuse macOS users)